Source/Extensions/Blazorise.DataGrid/Configuration/PagerOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Blazorise.DataGrid.Configuration
+{
+    /// <summary>
+    /// Defines the options for <see cref="DataGrid{TItem}"/> pager.
+    /// </summary>
+    public class PagerOptions
+    {
+        /// <summary>
+        /// Configures the pager buttons size.
+        /// </summary>
+        public Size Size { get; set; }
```

Comment:
Might it be better renamed to ButtonsSize? Or ButtonSize?

Reply:
Right, I actually debated that myself, but I guess I just though people could easily infer that it would do.
We can change. `ButtonSize` seems okay.